dev-util/bpftool/bpftool-7.5.0-r1.ebuild

```diff
@@ -27,14 +27,14 @@ S="${S_K}/tools/bpf/bpftool"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
-IUSE="caps +llvm"
+IUSE="caps llvm"
 
 RDEPEND="
-	sys-libs/binutils-libs:=
 	sys-libs/zlib:=
 	virtual/libelf:=
 	caps? ( sys-libs/libcap:= )
 	llvm? ( sys-devel/llvm:= )
```

Comment:
On reflection, this needs porting to llvm-r1.eclass too, but you can do that on a bump instead if you'd prefer.

Reply:
If by "porting to llvm-r1" you mean for the LLVM dependency then I'll do that on the next update.

Reply:
https://bugs.gentoo.org/938351